src/template.html

```diff
@@ -130,13 +130,24 @@
                 }).addTo(map);
             {{else}}
                 {{#if tileserverUrl}}
-                    L.tileLayer(
-                        '{{{tileserverUrl}}}',
-                        {
-                            maxZoom: maxZoom,
-                            fadeAnimation: false
-                        }
-                    ).addTo(map);
+                    {{#if tileSubdomains }}
+                        L.tileLayer(
+                            '{{{tileserverUrl}}}',
+                            {
+                                subdomains: {{{tileSubdomains}}},
+                                maxZoom: maxZoom,
+                                fadeAnimation: false
+                            }
+                        ).addTo(map);
+                    {{else}}
+                        L.tileLayer(
+                            '{{{tileserverUrl}}}',
+                            {
+                                maxZoom: maxZoom,
+                                fadeAnimation: false
+                            }
+                        ).addTo(map);
+                    {{/if}}
```

Comment:
make this if only in the subdomains part of the code
```suggestion
                        L.tileLayer(
                            '{{{tileserverUrl}}}',
                            {
                               {{#if tileSubdomains }}
                                subdomains: {{{tileSubdomains}}},
                                {{/if}}
                                maxZoom: maxZoom,
                                fadeAnimation: false
                            }
                        ).addTo(map);
```